qm-plugins/qm-object-cache/html/class-qm-output-object-cache.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Output class
+ *
+ * Class QM_Output_ObjectCache
+ */
+class QM_Output_ObjectCache extends QM_Output_Html {
+
+	public function __construct( QM_Collector $collector ) {
+		parent::__construct( $collector );
+
+		add_filter( 'qm/output/menu_class', array( $this, 'admin_class' ) );
+		add_filter( 'qm/output/menus', array( $this, 'admin_menu' ), 30 );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function name() {
+		return __( 'Object Cache', 'query-monitor' );
+	}
+
+
+	/**
+	 * Outputs data in the footer
+	 */
+	public function output() {
+		global $wp_object_cache;
+		if ( ! method_exists( $wp_object_cache, 'get_stats' ) ) {
+			echo '<div class="qm qm-non-tabular" id="' . esc_attr( $this->collector->id() ) . '">';
+			echo '<div id="object-cache-stats">';
+			$wp_object_cache->stats();
+			echo '</div></div>';
+			return;
+		}
+
+		$data = $this->collector->get_data();
+		$this->before_non_tabular_output();
+
+		$totals = $data['totals'] ?? false;
+		if ( $totals ) {
+			$this->output_before_section( 'Totals' );
+			if ( isset( $totals['query_time'] ) ) {
+				$this->output_table_row( 'Query Time', number_format_i18n( sprintf( '%0.1f', $totals['query_time'] * 1000 ), 1 ) . 'ms' );
+			}
+			if ( isset( $totals['size'] ) ) {
+				$this->output_table_row( 'Size', size_format( $totals['size'], 2 ) );
+			}
+			$this->output_after_section();
+		}
+
+		$stats = $data['stats'] ?? false;
+		if ( $stats ) {
+			$this->output_before_section( false );
+			foreach ( $stats as $stat => $value ) {
+				$this->output_table_row( $stat, $value );
```

Comment:
Would be nice to have these `$value` numbers formatted to include the locale's thousand separators, etc.

Reply:
I believe it is later down on L49 via `number_format_i18n()`?

Reply:
That's for the Query Time value in the first table. This line (L61) is for the numerical outputs in the second table (get, update, etc.), which isn't formatted.

The screenshot included in the PR description even shows `1888` unformatted.

Reply:
@GaryJones If you're referring to the `1888` value for `get_local`, those are the counts and thus, no formatting is needed.

Reply:
If a count is 4324987419041510 then it's going to be hard to read. Hence, formatting the number to use thousand-separators will make it easier.